frontend/app/components/import_list/import.component.ts

```diff
@@ -19,12 +21,15 @@ import { CsvExportService } from "../../services/csv-export.service";
 export class ImportComponent implements OnInit {
   public deletedStep1;
   public history;
+  public filteredHistory;
   public empty: boolean = false;
   public config = ModuleConfig;
   historyId: any;
   n_invalid: any;
   csvDownloadResp: any;
 
+  @ViewChild('search') search: any;
```

Comment:
Cette méthode fonctionne, mais on a pris l'habitude d'utiliser les Reactive Form d'angular pour la gestion des formulaire et des Inputs. Voir https://angular.io/guide/reactive-forms
```suggestion
  public search = new FormControl()
```